docs/quick_start/installation.rst

```diff
@@ -112,9 +112,9 @@ Three ``OCIO_INSTALL_EXT_PACKAGES`` options are available::
     cmake -DOCIO_INSTALL_EXT_PACKAGES=<NONE|MISSING|ALL>
 
 - ``NONE``: Use system installed packages. Fail if any are missing or 
-  don't meet minimum version requireements.
+  don't meet minimum version requirements .
```

Comment:
Looks like there is an extra space between `requirements` and `.` in both instances.